FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.Tool/Microsoft.Health.AnalyticsConnector.Tool.csproj

```diff
@@ -25,9 +25,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Update="appsettings-example-v1.json">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
     <None Update="appsettings.json">
```

Comment:
do we need to copy the `appsettings-example-v2.json` to the output?

Reply:
No, this is just an example for user on how to write the ConfigVersion 2 appsettings.